build.gradle.kts

```diff
@@ -95,7 +95,7 @@ tasks.register("buildActivityApiModel", GenerateTask::class) {
   apiPackage.set("uk.gov.justice.digital.hmpps.prisonerfromnomismigration.activities.api")
   configOptions.set(
     mapOf(
-      "dateLibrary" to "java8-localdatetime",
+      "dateLibrary" to "string",
```

Comment:
Interesting. I was expecting the localdatetime one but that always felt wrong because the description is "legacy only"!

Reply:
Well. ... This is a good one, the problem here is that there are no @JsonFormats in the generated dtos so the json created for the dates in the request dto is completely wrong. So reverted to conversion to string manually